ci-operator/step-registry/openstack/deprovision/deleteadditionalnetworks/openstack-deprovision-deleteadditionalnetworks-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
+  as: openstack-deprovision-deleteadditionalnetworks
+  from: openstack-installer
+  commands: openstack-deprovision-deleteadditionalnetworks-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+    - name: OS_CLOUD
+      default: "openstack"
+      documentation: Name of cloud to use from ${CLUSTER_PROFILE_DIR}/clouds.yaml file.
```

Comment:
SHARED_DIR, not CLUSTER_PROFILE_DIR.

CLUSTER_PROFILE_DIR indeed contains a clouds.yaml with one cloud named “openstack-cloud”, but we deprecated it.

Reply:
@adduarte I'm not sure if Pierre's comment was 100% addressed here, since he asked to use `SHARED_DIR`, unless I missed something?